changelog/changelog.py

```diff
@@ -228,6 +299,14 @@ def get_parser():
                                "as headers in the changelog)"
                           )
 
+    optional.add_argument("--header-labels",
+                          nargs="*",
```

Comment:
We probably don't want to allow an empty list of header labels if the option `--header-labels` is specified.
```suggestion
                          nargs="+",
```